apps/web/app/modules/Courses/Lesson/LessonItems/FillInTheBlanks/TextBlank.tsx

```diff
@@ -1,25 +1,70 @@
+import { cn } from "~/lib/utils";
+import { useEffect, useRef } from "react";
+
 type TextBlankProps = {
+  isQuiz: boolean;
   index: number;
-  studentAnswer?: string;
+  studentAnswer?: {
+    id: string;
+    optionText: string;
+    position: number | null;
+    isStudentAnswer?: boolean | null;
+    isCorrect?: boolean | null;
+    studentAnswerText?: string | null;
+  };
   handleOnBlur: (value: string, index: number) => void;
+  isQuizSubmitted?: boolean;
 };
 
 export const TextBlank = ({
+  isQuiz,
   index,
   studentAnswer,
   handleOnBlur,
+  isQuizSubmitted,
 }: TextBlankProps) => {
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (!inputRef?.current) return;
+
+    if (isQuiz) {
+      inputRef.current.value = studentAnswer?.studentAnswerText ?? "";
+    }
+  }, [isQuiz, isQuizSubmitted]);
```

Comment:
`React Hook useEffect has a missing dependency: 'studentAnswer?.studentAnswerText'. Either include it or remove the dependency array`

if it is intentional add the ignore line with short comment why